### DISCUSSION TITLE
Intel Discontinues High-Speed, Open-Source H.265/HEVC Encoder Project

### DISCUSSION THREAD

**Discussion by alatteri** [CESNET/UltraGrid#405] [General]
https://www.phoronix.com/news/Intel-Discontinues-SVT-HEVC

Not sure how this impacts its inclusion in libavcodec.

**Comment by MartinPulec** [comment]
Yeah, it is sad but could have been expected. On the other hand, the development was [far from tumultuous](https://github.com/OpenVisualCloud/SVT-HEVC/graphs/code-frequency). I don't think that SVT-HEVC will be included now as the [plugin](https://github.com/OpenVisualCloud/SVT-HEVC/tree/master/ffmpeg_plugin) has been out-of-tree for a long time, unless someone adopts the project ;)

From UG point of view we can it build (at least) for Linux as before. The patches will  diverge in future if the context of patched files changes but it can be fixed locally quite easily.

**Reply by alatteri** [reply]
Yup... haven't used SVT-HEVC in a long time, I just use x265 for everything.
